### ISSUE TITLE
Cannot build app with ^3.0.0-nullsafety.3

### ISSUE THREAD

**Issue by devj3ns** [miguelpruivo/flutter_file_picker#628]
**Describe the bug**

When trying to build an app with file_picker:  `^3.0.0-nullsafety.3` I get the following exception:

**Error Log**

```
/C:/flutter/.pub-cache/hosted/pub.dartlang.org/file_picker-3.0.0-nullsafety.3/lib/src/file_picker_result.dart:27:8: Error: A value of type 'List<String?>' can't be returned from a function with return type 'List<String>' because 'String?' is nullable and 'String' isn't.
 - 'List' is from 'dart:core'.
      .toList();
       ^
```

**Platform**
- [x] Android
- [x] iOS
- [x] Web
- [ ] Desktop (Go)

**Platform OS version**
Windows 10

**Flutter Version details**
[✓] Flutter (Channel stable, 2.0.0, on Microsoft Windows [Version 10.0.19042.804], locale de-DE)
    • Flutter version 2.0.0 at C:\flutter
    • Framework revision 60bd88df91 (17 hours ago), 2021-03-03 09:13:17 -0800
    • Engine revision 40441def69
    • Dart version 2.12.0

**Comment by osaxma** [comment]
Same issue here.  #626 should fix it once merged.

**Comment by miguelpruivo** [comment]
Fixed and published with 3.0.0-nullsafety.4

Thank you for catching it up. Mostly happened due to rebase conflict yesterday.
